.github/scripts/publish.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+set -e
+
+# List of packages to publish
+# CHANGE IT WHEN ADDING A NEW PROJECT TO BE RELEASED.
+PACKAGES=(
```

Comment:
we could generate PACKAGE from /package.json
maybe with something like `PACKAGES = jq -r '.workspaces.packages' package.json` excluding `'storybook'`

Reply:
Thanks for the suggestion ! 

I've implemented it and added a mechanism to exclude packages in the publish script.